Server/src/main/java/org/openas2/app/partner/AddPartnerCommand.java

```diff
@@ -17,57 +23,82 @@
  *
  * @author joseph mcverry
  */
+
 public class AddPartnerCommand extends AliasedPartnershipsCommand {
+
+    private Log logger = LogFactory.getLog(getClass().getName());
+
     public String getDefaultDescription() {
         return "Add a new partner to partnership store.";
     }
 
     public String getDefaultName() {
+
         return "add";
     }
 
     public String getDefaultUsage() {
+
         return "add name <attribute 1=value-1> <attribute 2=value-2> ... <attribute n=value-n>";
     }
 
     public CommandResult execute(PartnershipFactory partFx, Object[] params) throws OpenAS2Exception {
+
+        logger.info("partFx=" + partFx.toString());
+        // printStackTrace();
+
         if (params.length < 1) {
+
             return new CommandResult(CommandResult.TYPE_INVALID_PARAM_COUNT, getUsage());
         }
 
+        // * the original
         synchronized (partFx) {
 
             DocumentBuilder db = null;
             try {
                 db = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+
+                logger.info("Backing up " + db.getClass().getName());
+
             } catch (ParserConfigurationException e) {
                 throw new OpenAS2Exception(e);
             } catch (FactoryConfigurationError e) {
                 throw new OpenAS2Exception(e);
-            }
+            } 
+
+            
+        
 
             Document doc = db.newDocument();
+            logger.info("**** doc.getBaseUri()=" + doc.getBaseURI());
+            logger.info("********doc.toString()=" + doc.toString());
```

Comment:
These log staements are superfluous and adding this at INFO level logging will proliferate large log files. It should be at TRACE or DEBUG level and only if they actually provide real debug information.